lib/rubocop/cop/style/multiline_list.rb

```diff
@@ -0,0 +1,184 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks the line placement of the first element in a list of
+      # method parameters, method arguments, array elements, or hash elements.
+      #
+      # The cop start by looking for any of the following:
+      #
+      # * The first parameter in a method definition.
+      # * The first argument in a method call.
+      # * The first element in an array.
+      # * The first element in a hash.
+      #
+      # when the list of parameters, arguments or elements spans multiple lines
+      # It then ensures that the first such element does not also occur on the
+      # object's first line.
+      #
+      # It supports three configuration parameters:
+      #
+      # * EnableMethodParams - boolean
+      # * EnableMethodArgs - boolean
+      # * EnableHashElements - boolean
+      # * EnableArrayElements - boolean
+      #
+      # @example
+      #
+      #     # bad with EnableMethodParams
+      #     def method(foo, bar,
+      #         baz)
+      #       do_something
+      #     end
+      #
+      #     # good with EnableMethodParams
+      #     def method(
+      #         foo, bar,
+      #         baz)
+      #       do_something
+      #     end
+      #
+      #     # ignored with EnableMethodParams
+      #     def method foo,
+      #         bar
+      #       do_something
+      #     end
+      #
+      #     # bad with EnableMethodArgs
+      #     method(foo, bar,
+      #       baz)
+      #
+      #     # good with EnableMeethodArgs
+      #     method(
+      #       foo, bar,
+      #       baz)
+      #
+      #     # ignored with EnableMethodArgs
+      #     method foo, bar,
+      #       baz
+      #
+      #     # bad with EnableHashElements
+      #     { a: 1,
+      #       b: 2}
+      #
+      #     # good with EnableHashElements
+      #     {
+      #       a: 1,
+      #       b: 2 }
+      #
+      #     # bad with EnableArrayElements
+      #     [ :a,
+      #       :b]
+      #
+      #     # good with EnableArrayElements
+      #     [
+      #       :a,
+      #       :b]
+      #
+      class MultilineList < Cop
+        MSG = 'Move all elements below the first line of a multi-line list.'
```

Comment:
In the descriptions, and especially in this message, I think it would be more clear to talk about a preceding line break. Something like "Add a line break before the first element...".